flux_led/__init__.py

```diff
@@ -450,14 +450,16 @@ def __init__(self, ipaddr, port=5577, timeout=5):
         self.port = port
         self.timeout = timeout
 
+        self.protocol = None
```

Comment:
I thought about that but I hesitate because currently every parameter is defaulted so a user just wanting to set the protocol would have to know all the other parameters and their defaults just to pass in the 4th parameter. Another option could be to use kwargs but it would break existing users functionality.

I am using these changes in https://home-assistant.io/ and currently have the component in that system calling setProtocol after init if it's defined in the config.

```
bulb = flux_led.WifiLedBulb([IP_ADDRESS])
bulb.setProtocol("LEDENET")
```